File: 1352A/main.cpp
#include <iostream>
#include <vector>
#include <bits/stdc++.h>
using namespace std;
int main() {
    int t;
    cin >> t;
    while(t--){
        int n;
        cin >> n;

        vector<int> ans;
        int mul = 1;

        while(n>0){
            if(n%10!=0){
                ans.push_back((n%10)*mul);
            }
            n = n/10;
            mul = mul*10;
        }

        cout << ans.size() << endl;
        for(auto& i : ans){
            cout << i << " ";
        }
        cout << endl;
    }

    // for(auto& i:v){
    //     cout << i << endl;
    // }
    
    return 0;
}

File: 1352A/main.py
l=[]
t = int(input())
for i in range(t):
    n = int(input())
    mul = 1
    ans = []
    while(n>0):
        if(n%10!=0):
            ans.append((n%10)*mul)
        n = n//10
        mul = mul*10
    print(len(ans))
    for i in ans:
        print(i,end=" ")
# for i in l:
#     print(i)